workflow/pipeline/ephys.py

```diff
@@ -3,20 +3,22 @@
 
 from workflow import db_prefix
 from workflow.pipeline import lab, session
-from workflow.utils.paths import (get_ephys_root_data_dir,
-                                  get_processed_root_data_dir,
-                                  get_session_directory)
+from workflow.utils.paths import (
+    get_ephys_root_data_dir,
+    get_processed_root_data_dir,
+    get_session_directory,
+)
 
-__all__ = ['ephys', 'probe']
+__all__ = ["ephys", "probe"]
 
 
 # ------------- Activate "ephys" schema -------------
 
-SkullReference = lab.SkullReference
+BrainCoordinateReference = lab.BrainCoordinateReference
```

Comment:
```suggestion
SkullReference = lab.BrainCoordinateReference
```
The `element-array-ephys` expect a table named `SkullReference` in order to be activated